_posts/2018-01-04-notification-channels-android-o.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title:  "Notification Channels in Android Oreo"
+date:   2018-02-06 20:20:02
+description: Prepare your app for Android Oreo targets by implementing notification channels
+excerpt: "Prepare your app for Android Oreo targets by implementing notification channels"
+permalink: /posts/notification-channels-android-o
+tags: [android oreo, tutorial, notification channels, notification categories]
+---
+
+
+In (not-so) recent news, Google is going to be enforcing that apps are targetted for SDK 26 (Android Oreo) by late 2018, in order to ensure security and improved performance in apps. You can read more about the announcement [here](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
```

Comment:
"enforce that all apps target API Level 26"

Reply:
🔒